logger/logger_test.go

```diff
@@ -223,3 +223,19 @@ func TestAddContext(t *testing.T) {
 	assertLogFormatAndCompareContent(t, string(buf.Bytes()),
 		kv.FormatLog("logger-tester", kv.Info, "2", M{"a": "b"}))
 }
+
+func TestFailAddReservedContext(t *testing.T) {
+	logger := New("logger-tester")
+	reservedKeyNames := map[string]bool{
+		"title":  true,
+		"source": true,
+		"value":  true,
+		"type":   true,
+		"level":  true,
+	}
+	for k := range reservedKeyNames {
+		res := updateContextMapIfNotReserved(logger.globals, k, "test")
+		msg := "Should not be able to set key " + k
+		assert.False(t, res, msg)
```

Comment:
Nit. Perhaps instead of using a return value, the code could check `logger.globals` directly?  That would also remove the need for `_ =` where `updateContextMapIfNotReserved` is used.